frontend/Makefile

```
@@ -40,6 +40,9 @@ release-build:
 ##################################################
 # Local development
 ##################################################
+build: # Build the Next.js local dev server in Docker
+	docker-compose build --no-cache nextjs
```

```suggestion
	docker compose build --no-cache nextjs storybook
```
docker compose doesn't get a dash anymore, I also type it accidentally all the time.
let's add storybook here too, unless you want to have separate build-dev and build-storybook make commands.
can you also update our renovate review instructions with this make command [here](https://app.gitbook.com/o/cFcvhi6d0nlLyH2VzVgn/s/v1V0jIH7mb7Yb3jlNrgk/engineering/draft-renovate-pr-playbook-and-workflow/~/comments?context=post&node=d827e994cee6430dbbbdcc257800ff64#:~:text=%3CCtrl%3E%20c-,run%20docker%20compose%20build%20%2D%2Dno%2Dcache%20nextjs%20storybook%20to%20recreate%20all%20layers%20of%20your%20docker%20build%20image%20for%20the%20frontend%20app%20and%20storybook,-run%20make%20%2DC)

---

I propose splitting splitting up the app and storybook to make it more modular